src/Data/Unfoldable1.purs

```diff
@@ -0,0 +1,73 @@
+module Data.Unfoldable1
+  ( class Unfoldable1, unfoldr1
+  , replicate1
+  , replicate1A
+  , singleton
+  , range
+  ) where
+
+import Prelude
+
+import Data.Maybe (Maybe(..))
+import Data.Traversable (class Traversable, sequence)
+import Data.Tuple (Tuple(..))
+
+-- | This class identifies non-empty data structures which can be _unfolded_,
+-- | generalizing `unfoldr1` on non-empty lists or arrays.
```

Comment:
`unfoldr1` is not provided elsewhere, so maybe nix that part of the comment?

Reply:
Hmm, good point, `unfoldr` is not provided elsewhere either so I guess that applies to both modules.

Reply:
Must have been copy-paste from some Haskell?